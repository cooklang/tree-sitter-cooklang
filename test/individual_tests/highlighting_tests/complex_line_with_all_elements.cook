In #large pot{}, add @olive oil{2%tbsp} and heat ~{3%minutes} -- don't overheat!