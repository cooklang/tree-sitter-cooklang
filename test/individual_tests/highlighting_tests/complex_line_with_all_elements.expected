(recipe
  (step
    (plain_text)
    (cookware
      name: (cookware_text))
    (plain_text)
    (ingredient
      name: (ingredient_text)
      (amount
        (quantity)
        (units)))
    (plain_text)
    (timer
      (amount
        (quantity)
        (units)))
    (plain_text)
    (comment)))